Pages/Hotels/Sprint2Page.js

```diff
@@ -0,0 +1,278 @@
+        await this.commands.clickWebElement(this.privateResidence)
+    }
+    async verifyStep1Of3IsDisplayed(){
+        return await this.commands.isElementDisplayed(this.step1Of3)
+    }
+    async verifySeeHowMuch(){
+        return await this.commands.isElementDisplayed(this.seeHowMuchYouCouldEarn)
+    }
+    async clickIncreaseBedrooms(number){
+        for(let i = 0; i < number; i++){
+            await this.commands.clickWebElement(this.increaseBedrooms)
+        }
+    }
+    async clickIncreaseBathrooms(number){
+        for(let i = 0; i < number; i++){
+            await this.commands.clickWebElement(this.increaseBathrooms)
+        }
+    }
+    async clickNextButton(){
+        await this.commands.clickWebElement(this.nextButton)
+    }
+    async verifyStep2Of3IsDisplayed(){
+        return await this.commands.isElementDisplayed(this.step2Of3)
+    }
+    async verifyWhereIsYourPropertyIsDisplayed(){
+        return await this.commands.isElementDisplayed(this.whereIsYourPropertyLocated)
+    }
+    async enterAddress(destination) {
+        await this.commands.clickWebElement(this.enterAddressLocator);
+        await this.commands.typeInWebElement(this.enterAddressLocator, destination)
+        await browser.pause(2000);
+    }
+    async fromAutoSuggestion(selectThis) {
+        await this.commands.selectAddressFromAutoSuggestion(this.addressAutoSuggestionLocator, selectThis);
+        await browser.pause(2000)
+    }
+    async verifyMapIsDisplayed(){
+        return await this.commands.isElementDisplayed(this.map)
+    }
+    async verifyPinIsDisplayed(){
+        return await this.commands.isElementDisplayed(this.mapPin)
+    }
+    async clickFiveStar(){
+        await this.commands.scroll(this.fiveStarButton)
+        await this.commands.clickWebElement(this.fiveStarButton)
+    }
+    async clickPriceFromSortByDropdown(){
+        await this.commands.clickWebElement(this.sortBy)
+        await browser.pause(2000)
+        //await $(this.sortBy).selectByVisibleText(this.price)
+        await this.commands.clickWebElement(this.price)
+        await browser.pause(2000)
+    }
+    async getSelectedFiveStarRating(){
+        let fiveStarText = await this.commands.getElementText(this.fiveStarButton)
+        let fiveStarNumber = parseInt(fiveStarText);
+        console.log(typeof fiveStarNumber);
+        console.log('THIS IS THE NUMBER: ' + fiveStarNumber);
+        return fiveStarNumber;
+    }
+    async verifyHotelsAreRatedAsSelected(){
+        let fiveStarRating = await this.commands.getElementText(this.fiveStarRating)
+        let fiveStarNumber = parseInt(fiveStarRating);
+        console.log(typeof fiveStarNumber)
+        console.log('THIS IS THE NUMBER: ' + fiveStarNumber);
+        return fiveStarNumber;
+    }
+    async getStarRating(){
+        let starsArray = [];
+        let starsObjects = await $$(this.fiveStarRating)
+        for(const star of starsObjects){
+            console.log("THIS IS AN ELEMENT: ")
```

Comment:
You can print anything locally for your info or for debugging purpose but not required to push that. In professional project we don't print anything. No one looks at log but the check report.